src/conn/conn_layered.c

```diff
@@ -372,11 +375,18 @@ __wt_layered_table_manager_add_table(
       __wt_atomic_load32(&manager->state) == WT_LAYERED_TABLE_MANAGER_RUNNING,
       "Adding a layered table, but the manager isn't running");
     __wt_spin_lock(session, &manager->layered_table_lock);
+
+    if (ingest_id >= manager->open_layered_table_count) {
+        WT_ERR(__wt_realloc_def(session, &manager->entries_allocated,
+          manager->open_layered_table_count * 2, &manager->entries));
```

Comment:
I'm guessing it works or the test would fail, but when does `open_layered_table_count` first get set to something other than 0?
Would it be better to replace `manager->open_layered_table_count * 2` with `ingest_id * 2`? and/or assert that the open count is non-zero?

Reply:
It's first set to
```
manager->open_layered_table_count = conn->next_file_id + 1000;
```
Yes, I can change that.